integration-tests/maven/src/test/java/io/quarkus/maven/it/NativeAgentIT.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.maven.it;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.maven.shared.invoker.MavenInvocationException;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.maven.it.verifier.MavenProcessInvocationResult;
+import io.quarkus.maven.it.verifier.RunningInvoker;
+
+@EnableForNative
+public class NativeAgentIT extends MojoTestBase {
+
+    @Test
+    public void testRunIntegrationTests() throws MavenInvocationException, IOException, InterruptedException {
+        final File testDir = initProject("projects/native-agent-integration");
+        final RunningInvoker running = new RunningInvoker(testDir, false);
+
+        MavenProcessInvocationResult runJvmITsWithAgent = running.execute(
+                List.of("clean", "verify", "-DskipITs=false", "-Dquarkus.test.integration-test-profile=test-with-native-agent"),
+                Map.of());
+        assertThat(runJvmITsWithAgent.getProcess().waitFor()).isZero();
+
+        MavenProcessInvocationResult runNativeITs = running
+                .execute(List.of("verify", "-Dnative", "-Dquarkus.native.agent-configuration-apply"), Map.of());
```

Comment:
See https://gist.github.com/galderz/34785597c77893b137f2e47fad1de394

Reply:
Good, thanks.

I think it could be handy to print the mvn commands so that on failures we can copy paste them to reproduce and inspect the failure.

Reply:
Couldn't see an easy way to achieve just this, but instead I added some `INFO` logging to Maven process invoker so that it prints out the command being executed.